rpc/handlers.go

```diff
@@ -480,4 +509,16 @@ func (h *Handler) MethodsV0_6() ([]jsonrpc.Method, string) { //nolint: funlen
 			Handler: h.Unsubscribe,
 		},
 	}, "/v0_6"
+
+}
+func RegisterHandlers(rpcServer *jsonrpc.Server, h *Handler) error {
```

Comment:
The list of methods defined above are registered elsewhere in the code, no need for this func :)